libs/community/langchain_community/vectorstores/lancedb.py

```diff
@@ -681,7 +681,10 @@ def delete(
         if filter:
             tbl.delete(filter)
         elif ids:
-            tbl.delete(f"{self._id_key} in ('{{}}')".format(",".join(ids)))
+            quoted_ids = [f"'{id_val}'" for id_val in ids]
```

Comment:
Would you mind sanitizing the ids for `'` values? I don't think the ability to escape sql on ids is expected here